FAILURE: Build failed with an exception.

I get the following error when I run './gradlew build' inside my /bisq-docs folder.
I used this script to install JAVA: https://github.com/bisq-network/bisq/blob/master/scripts/install_java.sh
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':asciidoctor'.
> Could not resolve all files for configuration ':asciidoctor'.
   > Could not resolve org.codehaus.groovy:groovy-all:2.3.9.
     Required by:
         project : > org.asciidoctor:asciidoctorj-groovy-dsl:1.0.0.Alpha2
      > Could not resolve org.codehaus.groovy:groovy-all:2.3.9.
         > Could not get resource 'https://jcenter.bintray.com/org/codehaus/groovy/groovy-all/2.3.9/groovy-all-2.3.9.pom'.
            > Could not GET 'https://jcenter.bintray.com/org/codehaus/groovy/groovy-all/2.3.9/groovy-all-2.3.9.pom'.
               > sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
1 actionable task: 1 executed
```

@m52go pointed me to [this thread](https://stackoverflow.com/questions/21076179/pkix-path-building-failed-and-unable-to-find-valid-certification-path-to-requ) but it's beyond my understanding.

--- a/build.gradle
+++ b/build.gradle
@@ -1,5 +1,5 @@
 plugins {
-    id 'org.asciidoctor.convert' version '1.5.7'
+    id 'org.asciidoctor.convert' version '2.3.0'
 }
 
 asciidoctor {

--- a/css/bisq.css
+++ b/css/bisq.css
@@ -1200,6 +1200,10 @@ a {
     margin-bottom: 2.25em;
 }
 
+#header #revnumber {
+    display: none;
+}
+
 @media only screen and ( max-width: 768px ) {
     #toc img {
         margin-top: 40px;

